internal/pkg/deploy/cloudformation/project.go

```diff
@@ -63,6 +63,70 @@ func (cf CloudFormation) CreateProjectResources(project *archer.Project) error {
 	return nil
 }
 
+// GetProjectResourcesByRegion fetches all the regional resources for a particular region.
```

Comment:
I suppose this should be used by the `deploy` command, right?

Reply:
That and package - to get the ECR repo.